tests/test_aws.py

```diff
@@ -1043,6 +1043,27 @@ def test_retrieve_subject_token_success_environment_vars(self, utcnow, monkeypat
             }
         )
 
+    @mock.patch("google.auth._helpers.utcnow")
+    def test_retrieve_subject_token_success_environment_vars(self, utcnow, monkeypatch):
```

Comment:
This test has the same name as the above test.

Reply:
done